### PR TITLE
Maintain | Remove usage of deprecated Symfony Templating `EngineInterface`

### PR DESCRIPTION
**Pull request by stloyd**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | yes
| Deprecations?   | no
| License         | MIT

Part of: https://github.com/Sylius/Sylius/issues/13274

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php [hunk 1]
```diff
@@ -29,29 +27,21 @@ final class DashboardController
 {
     private ChannelRepositoryInterface $channelRepository;
 
-    /** @var EngineInterface|Environment */
-    private $templatingEngine;
+    private Environment $templatingEngine;
 
     private RouterInterface $router;
 
-    private ?SalesDataProviderInterface $salesDataProvider;
```
Comment (stloyd) [inline src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php:37]: Marked as never used.

**Comment by stloyd** [comment]
@lchrusciel anything blocking this?

**Review by vvasiloi — APPROVED** [review]
I don't know what @lchrusciel has to say, but this looks good to me.

**Review by loic425 — APPROVED** [review]

**Comment by lchrusciel** [comment]
Hey Stephane,

sorry, it took so long. @loic425 picked up migration to Symfony6 so we finally can focus on it. Do you think, it is possible to rebase the branch and resolve my comment? Or @loic425 can you pick it up?

**Comment by loic425** [comment]
@lchrusciel When trying to rebase it, I saw no needed changes...
https://github.com/Sylius/Sylius/compare/master...loic425:fix/engine?expand=1

**Comment by stloyd** [comment]
Closing as this was implemented in https://github.com/Sylius/Sylius/pull/13709

**Pull request closed without merging**